src/api/subscriptions.js

```diff
@@ -359,6 +359,11 @@ export const ShowcaseEvent = {
   EVENT_SHOWCASE_INELIGIBLE_PAYWALL: 'EVENT_SHOWCASE_INELIGIBLE_PAYWALL', // When the user is not eligible for showcase entitlements
   EVENT_SHOWCASE_NO_ENTITLEMENTS_PAYWALL:
     'EVENT_SHOWCASE_NO_ENTITLEMENTS_PAYWALL', // When the user has no remaining showcase entitlements
+
+  // Events indicating the user interacted with the Showcase Regwall
```

Comment:
Just checking if these need to be here?

All of the events related to the meter toast are just in the API messages: https://github.com/subscriptions-project/swg-js/blob/c10f2949cefdc476fef5faa49a348cb75475568a/src/proto/api_messages.js#L85-L89

Reply:
Good call! It's simpler now